beater/config/config.go

```diff
@@ -42,6 +44,45 @@ var (
 	regexObserverVersion = regexp.MustCompile("%.*{.*observer.version.?}")
 )
 
+type kibanaConfig struct {
+	Enabled bool                `config:"enabled"`
+	Config  kibana.ClientConfig `config:",inline"`
+}
+
+func (k *kibanaConfig) Unpack(c interface{}) error {
```

Comment:
Would it make sense to make this `Unpack(cfg *ucfg.Config)` instead? i.e.

```go
func (k *kibanaConfig) Unpack(cfg *ucfg.Config) error {
    if err := cfg.Unpack(&k.Config); err != nil {
        return err
    }
    k.Enabled = cfg.Enabled()
    k.Config.Host = strings.TrimRight(k.Config.Host, "/")
    return nil
}
```

IIANM, that'll save a round-trip from Config -> interface{} -> Config

Reply:
@axw Another great suggestion, thanks.  I had to use `common.Config` to get the `Enabled()` method - would you double check 99decba9649b831ae79aec3464bdb8826eca8b01 to see if it looks right?